### DISCUSSION TITLE
Unable to push via Obsidian commands

### DISCUSSION THREAD

**Discussion by deshjackson** [Vinzent03/obsidian-git#280] [Problems] [answered]
*Apparent duplicate, but I believe this issue is different than the previous thread.*

I'm getting this error in the console when running the git push Obsidian command: 
<img width="959" alt="Screen Shot 2022-08-19 at 12 33 33 PM" src="https://user-images.githubusercontent.com/22530702/185675580-6fbd8d94-98a3-499d-90f7-74ca60fc8ce4.png">

Code block referenced in the error:
<img width="1015" alt="Screen Shot 2022-08-19 at 12 35 09 PM" src="https://user-images.githubusercontent.com/22530702/185675773-6b95f7f1-8dde-43b3-8dc2-b323ef424541.png">

Commits, branching, and other commands work through Obsidian commands, and of course I'm able to push/pull from my own terminal. I haven't seen any signs of this issue in the forums/etc., so maybe I'm doing some thing wrong on my side, but posting just in case. 

Thanks!

**Comment by Vinzent03** [comment, marked as answer]
Should be fixed in the new release.
